ansible-core/packaging/debian/rules

```diff
@@ -1,9 +1,14 @@
 #!/usr/bin/make -f
-# -- makefile --
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: Ansible Project, 2020
 
-DEB_PYTHON3_MODULE_PACKAGES=ansible-core ansible_test
-#DEB_PYTHON_INSTALL_ARGS_ALL="--install-purelib=/usr/lib/python2.7/site-packages/"
-DEB_PYTHON_DISTUTILS_INSTALLDIR_SKEL = /usr/lib/python3/dist-packages/
+export PYBUILD_NAME = ansible-core
+export PYBUILD_DISABLE = test
```

Comment:
Have you looked into running tests?

Reply:
no, not after we disabled them in the name of progress 🤣 I'll add an issue for this